src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Peering/AzureExpressRouteCircuitPeeringConfigBase.cs

```diff
@@ -48,7 +48,7 @@ public class AzureExpressRouteCircuitPeeringConfigBase : NetworkBaseCmdlet
             Mandatory = true,
             HelpMessage = "The PeerAsn")]
         [ValidateNotNullOrEmpty]
-        public int PeerASN { get; set; }
+        public uint PeerASN { get; set; }
```

Comment:
@pashukl have we validated that the server properly handles the new range of values when moving from `int` to `uint`?

Reply:
@cormacpayne : This is a breaking change, but the changes in SDK and Swagger should have taken care of it already.  

 I am looking into this.

Reply:
@pashukl yes, you can regenerate the markdown help files using `platyPS`. Please refer to the following document: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md

Reply:
@pashukl You can also edit examples/descriptions manually.

Reply:
@cormacpayne @maddieclayton : The examples/descriptions have been edited manually.  Am I missing anything else ?